environments/pull-environments.sh

```diff
@@ -27,9 +27,12 @@ while getopts ':r:i:e:' OPTION; do
   e)
     exclude="$OPTARG"
     ;;
+  v)
+    version="$OPTARG"
+    ;;
 
   ?)
-    echo "Usage: $(basename $0) [-r region] [-i include] [-e exclude]"
+    echo "Usage: $(basename $0) [-r region] [-i include] [-e exclude] [-v version]"
```

Comment:
Why do we need to add a version flag to the legacy pull command?

Reply:
This the script to pull all the environments.
```sh
if [ "$version" == "gen2" ]; then
  printf $dirs | xargs -P $numParallelTasks -I {} sh -c ""$shell_path"/pull-environment-gen2.sh {}"
else
  printf $dirs | xargs -P $numParallelTasks -I {} sh -c ""$shell_path"/pull-environment.sh {} "$region""
fi
```
`pull-enviroment.sh` and `pull-enviroment-gen2.sh` are used to pull a single environment for gen1 and gen2 respectively.